doc/man7/evp.pod

```diff
@@ -13,19 +13,19 @@ evp - high-level cryptographic functions
 The EVP library provides a high-level interface to cryptographic
 functions.
 
-L<B<EVP_Seal>I<...>|EVP_SealInit(3)> and L<B<EVP_Open>I<...>|EVP_OpenInit(3)>
-provide public key encryption and decryption to implement digital "envelopes".
+The L<B<EVP_Seal>I<XXX>|EVP_SealInit(3)> and L<B<EVP_Open>I<XXX>|EVP_OpenInit(3)>
```

Comment:
Hmmm, can I suggest something other than `XXX`?  How about `OP` (for operation), or more explicitely (using bash syntax) `{Init,Update,Final}`?

Reply:
Well, I just stumbled over the fact that conventions in evp.pod and x509.pod were inconsistent. If you prefer, I can follow your suggestion. Do you have similar suggestions for x509.pod?

Reply:
Inconsistent?  They both currently use ellipses, I'm not sure how that's inconsistent...

For x509.pod, it's more difficult for some functions.  For example, the `X509_` ones is a huge range, not easily summarized in one word.

Actually, this gives me second thoughts on this PR...

Reply:
> Inconsistent? They both currently use ellipses, I'm not sure how that's inconsistent...

The ellipses were changed in `x509.pod` to `XXX` in #4924. 

> Actually, this gives me second thoughts on this PR...

If you have different ideas about how it should be done, you may feel free to take over my PR or create a new one. I have no problems if you do so.

Reply:
Ahhhh, I hadn't noticed that change.  In that case, ignore my comments.

Reply:
Your suggestion to use bash style syntax `{Init,Update,Final}` sounded reasonable. If you like, I can still do it. But not before this evening or so.

Reply:
Well, that's doable for a short set of functions that isn't expected to change much over time.  I challenge you to do the same for `X509_...` ;-)

Reply:
No, I meant to follow your proposal for evp.pod and leave everything in x509.pod as it is.

Reply:
Or just merge this PR, as you prefer.

Reply:
I think leave it as it is now.  Consistency trumps my personal desires...